### PR TITLE
Make minimum changes to get module working

### PR DESCRIPTION
**Pull request by tf3**
Fixes https://trello.com/c/gJjd0u0u/1-make-module-usable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,23 @@
...
 
-### Installing Dependencies
+## Setup (local)
 
-From within the root directory:
+1. Install dependencies: `npm install`
+2. Build code: `npm run build`
+3. Make sure you have MongoDB installed and running.
+4. Seed database: `npm run seed`
+5. Start server: `npm start`
```
Comment (garrowat) [inline README.md:19]: Nice addition!

File: package.json [hunk 2]
```diff
@@ -41,8 +41,7 @@
     "react-dom": "^16.8.6",
     "react-modal": "^3.8.2",
     "styled-components": "^4.3.2",
-    "supertest": "^4.0.2",
-    "webpack": "^4.35.3"
+    "supertest": "^4.0.2"
```
Comment (garrowat) [inline package.json:44]: Not sure what supertest is used for, but looks good to me

**Review by garrowat — APPROVED** [review]
Everything looks good to me

File: package.json (hunk 2 above)
Comment (glkapitan) [reply package.json:44]: Looks good.  We used supertest to test api end points.

**Review by glkapitan — COMMENTED** [review]
Looks good

**Pull request merged**